Tests/LibWeb/Fixtures/http-test-server.py

```diff
@@ -22,10 +22,10 @@ class Echo:
     method: str
     path: str
     status: int
-    headers: Dict[str, str] | None
-    body: str | None
-    delay_ms: int | None
-    headers: dict | None
+    headers: Optional[Dict[str, str]]
+    body: Optional[str]
+    delay_ms: Optional[str]
```

Comment:
Was this change from int to str intended? 👀

Reply:
Ah, it was not! That property isn't used right now anyway, but if you want to submit a patch for that, it'd be welcome :)